src/configure/configure.ts

```diff
@@ -166,7 +166,9 @@ class PipelineConfigurer {
             await this.createGithubServiceConnection();
         }
 
-        await this.createAzureRMServiceConnection();
+        if(!!this.inputs.pipelineParameters.pipelineTemplate.targetType && !!this.inputs.pipelineParameters.pipelineTemplate.targetKind) {
```

Comment:
checking for target type should be sufficient here.

Reply:
Actually we should check for TargetKind, because Targettype is supposed to be None